test/patterns/bidirect_index/test_bidirect_index.py

```diff
@@ -1,69 +1,68 @@
 # SPDX-FileCopyrightText: Copyright (c) 2019-2025 Aibolit
 # SPDX-License-Identifier: MIT
-
 import os.path
 from pathlib import Path
 from unittest import TestCase
-import unittest
+
 from aibolit.patterns.bidirect_index.bidirect_index import BidirectIndex
 
 
-@unittest.skip('Not implemented')
 class BidirectIndexTestCase(TestCase):
     dir_path = Path(os.path.realpath(__file__)).parent
 
     def test_bidirect_index_increase_decrease(self):
+        file_path = Path(self.dir_path, 'BidirectIndexIncreaseDecrease.java')
         self.assertEqual(
-            BidirectIndex().value(Path(self.dir_path, 'BidirectIndexIncreaseDecrease.java')),
+            BidirectIndex().value(file_path),
```

Comment:
I suggest that we leave the changes in the test files outside of the present PR (apparently, except making the test non-skipped). It is OK to do it in another one, dedicated to the refactoring if necessary.